CMake/External_VTK.cmake

```diff
@@ -54,13 +54,31 @@ set(vtk_cmake_args ${vtk_cmake_args}
   )
 
 # Qt
-add_package_dependency(
-  PACKAGE VTK
-  PACKAGE_DEPENDENCY Qt
-  PACKAGE_DEPENDENCY_ALIAS Qt4
-)
-if(QT_QMAKE_EXECUTABLE)
-  set(BUILD_QT_WEBKIT ${QT_QTWEBKIT_FOUND})
+if(Qt_version_major EQUAL 4)
+  add_package_dependency(
+    PACKAGE VTK
+    PACKAGE_DEPENDENCY Qt
+    PACKAGE_DEPENDENCY_ALIAS Qt4
+  )
+  if(QT_QMAKE_EXECUTABLE)
+    set(BUILD_QT_WEBKIT ${QT_QTWEBKIT_FOUND})
+    set(vtk_cmake_args ${vtk_cmake_args}
+      -DVTK_RENDERING_BACKEND:STRING=OpenGL2
```

Comment:
If we're compiling the latest VTK, the rendering backend should be OpenGL2 irrespective of the Qt version.

Reply:
I don't believe that works, at least for ViViA.